packages/gasket-cli/src/scaffold/actions/read-config.js

```diff
@@ -0,0 +1,89 @@
+const { pluginIdentifier, flattenPresets } = require('@gasket/resolve');
```

Comment:
I don't think this file is necessary. You've duplicated logic from global-prompts, but notice in the originals, it won't actually trigger a prompt if the setting is found in the context.

As long as you've applied the provided `--config` (or file) to the create context before executing the global prompts action, it should be able to handle it.